cmd/bosun/sched/views.go

```diff
@@ -62,9 +62,13 @@ type IncidentSummaryView struct {
 	CritNotificationChains [][]string
 }
 
-func MakeIncidentSummary(c *conf.Conf, s SilenceTester, is *models.IncidentState) IncidentSummaryView {
-	warnNotifications := c.Alerts[is.AlertKey.Name()].WarnNotification.Get(c, is.AlertKey.Group())
-	critNotifications := c.Alerts[is.AlertKey.Name()].CritNotification.Get(c, is.AlertKey.Group())
+func MakeIncidentSummary(c conf.RuleConfProvider, s SilenceTester, is *models.IncidentState) IncidentSummaryView {
+	alert := c.GetAlert(is.AlertKey.Name())
+	if alert == nil {
+		panic("should not be called on nonexistant alert, for now at least TODO?")
```

Comment:
I hate panics. Return an err maybe? Return nil maybe? Is this only called from web requests?